server/render/index.js

```diff
@@ -96,7 +96,9 @@ export function serverRender( req, res ) {
 
 	context.head = { title, metas, links };
 
-	if ( config.isEnabled( 'desktop' ) ) {
+	if ( context.template && typeof context.template === 'string' ) {
+		res.render( context.template, context );
+	} else if ( config.isEnabled( 'desktop' ) ) {
```

Comment:
(You can drop these changes if you decide to go with my [suggestion](https://github.com/Automattic/wp-calypso/pull/11475/commits/8484466d553ad2d2cfed66954a337cecf8a9aeaa#r102709845).)